src/core/services/compiler/compiler.js

```diff
@@ -10,74 +10,91 @@ angular
 
 /**
  * @ngdoc service
- * @name $mdCompiler
+ * @name $mdCompilerProvider
  * @module material.core.compiler
  * @description
- * The $mdCompiler service is an abstraction of AngularJS's compiler, that allows developers
- * to easily compile an element with options like in a Directive Definition Object.
+ * The `$mdCompiler` is able to respect the AngularJS `$compileProvider.preAssignBindingsEnabled` state when using
+ * AngularJS versions between 1.5.10 and 1.7.0.
+ * See the [AngularJS documentation for `$compile.preAssignBindingsEnabled`
```

Comment:
`$compile` --> `$compileProvider`

Reply:
Doh! Not sure how I missed those, thanks.